src/lib/ui/Snackbar/Snackbar.tsx

```diff
@@ -29,29 +30,41 @@ export function Snackbar({
   autoDismiss = false,
   dismissAfterMs = 5000,
   disableCloseButton = false,
+  automaticVisibility = false,
   onClose = () => {},
 }: SnackbarProps): JSX.Element {
+  const [hidden, setHidden] = useState(visible)
+
+  useEffect(() => {
+    setHidden(!visible)
+  }, [visible])
```

Comment:
Isn't this triggering a linter warning? 

```suggestion
  }, [visible, setHidden])
```